flytekit-local-engine/src/test/java/org/flyte/localengine/LocalEngineTest.java

```diff
@@ -556,18 +556,21 @@ public TestCaseExhaustivenessWorkflow() {
     }
 
     @Override
-    public void expand(SdkWorkflowBuilder builder) {
-      SdkBindingData<Long> x = builder.inputOfInteger("x");
+    public NoOpType expand(SdkWorkflowBuilder builder, NoOpType input) {
```

Comment:
```suggestion
    public NoOpType expand(SdkWorkflowBuilder builder, NoOpType input) {
      SdkBindingData<Long> x = input.x();
```
Doing the change this way simplifies the code bellow